test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -163,7 +163,7 @@ def test_ecs_pytorch_training_dgl_gpu(
     if Version(image_framework_version) == Version("1.6") and image_cuda_version == "cu110":
         pytest.skip("DGL does not suport CUDA 11 for PyTorch 1.6")
     # TODO: Remove when DGL gpu test on ecs get fixed
-    if Version(image_framework_version) == Version("1.10") and image_cuda_version == "cu113":
+    if Version("1.10") <= Version(image_framework_version) < Version("1.11") and image_cuda_version == "cu113":
```

Comment:
Please change this to `Version(image_framework_version) in SpecifierSet("==1.10.*")`
```suggestion
    if Version(image_framework_version) in SpecifierSet("==1.10.*") and image_cuda_version == "cu113":
```